packages/plugin-events-manager/src/commands/event-create.js

```
@@ -0,0 +1,129 @@
+		new ActionRowBuilder().addComponents(titleInput),
+		new ActionRowBuilder().addComponents(descriptionInput),
+		new ActionRowBuilder().addComponents(dateTimeInput),
+		new ActionRowBuilder().addComponents(locationInput),
+		new ActionRowBuilder().addComponents(maxAttendeesInput)
+	]
+
+	modal.addComponents(...rows)
+	return modal
+}
+
+function createEventEmbed(eventData, creator) {
```

ESLint

[error] 'createEventEmbed' is defined but never used.

(@typescript-eslint/no-unused-vars)